### PR TITLE
Add my profile page

### PR DESCRIPTION
**Pull request by JulioAPeraza**
I added a post and a photo to team/_posts and assets/images/team respectively.

### CODE REVIEW

File: team/_posts/2017-05-14-peraza-julio.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+github: JulioAPeraza
+orcid: 0000-0003-3816-5903
+osf:
+publons:
+researchgate:
+impactstory:
+scholar: 3mxI9IIAAAAJ&hl
+site: https://julioaperaza.github.io
+twitter:
+---
```
Comment (tsalo) [inline team/_posts/2017-05-14-peraza-julio.md:24]: Can you add a blurb about yourself here?

**Review by tsalo — CHANGES_REQUESTED** [review]

File: team/_posts/2017-05-14-peraza-julio.md (hunk 1 above)
Comment (JulioAPeraza) [reply team/_posts/2017-05-14-peraza-julio.md:24]: ok

**Comment by tsalo** [comment]
LGTM. Is this ready for me to merge?

**Comment by JulioAPeraza** [comment]
Done.
Thank you.

**Pull request merged**